tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "sourceMap": true
```

Comment:
add `"noImplicitAny": true,` and start use types

Reply:
done